UKHomeOffice/nmsw-ui
# NMSW-1081 View UPT/ETA summary at First-time log-in

# Ticket

NMSW-1081

----

## AC

As a guest admin
I want to view detailed ETA information the FIRST time I log in
So that I understand the system changes

----

## To test

- run app
- sign in
- > You should be taken to ETA info page
- click continue
- > you should be taken to the dashboard
- > you should see an etaCookie in application in devtools
- sign out
- sign back in
- > you should be taken to dashboard (should not see the ETA page)
- delete the eta cookie
- sign out
- sign back in
- > you should see the ETA page again

----

## Notes

- There is a missing email on the do not board status info as we do not know the address the email will be sent from
